src/components/ReviewPage.jsx

```diff
@@ -1,22 +1,43 @@
 import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
-import { getReviewById } from "../api";
+import { getReviewById, updateReviewById } from "../api";
 import Comments from "./Comments";
+import Loading from "./Loading";
+import Modal from "./Modal";
+
 const commentIcon = require('../assets/chat.png');
 const likeIcon = require('../assets/heart.png');
 
 function ReviewPage() {
     const [review, setReview] = useState({});
+    const [voteCount, setVoteCount] = useState('')
     const [reviewLoading, setReviewLoading] = useState(true);
+    const [error, setError] = useState('');
+    const configuredDate = new Date(review.created_at).toDateString()
     const {review_id} = useParams();
     useEffect(() => {
         getReviewById(review_id).then(reviewFromAPI => {
             setReview(reviewFromAPI[0])
             setReviewLoading(false)
+            setVoteCount(reviewFromAPI[0].votes)
         })
     }, [review_id])
-    const configuredDate = new Date(review.created_at).toDateString()
-    if (!reviewLoading) {
+    
+    let vote = {
+        inc_votes: 1
+    }
+ 
+    const handleVote = (num) => {
+        vote.inc_votes = num
```

Comment:
Ideally, you should change the state before the api call, as that way you are optimistically rendering the desired result. The catch block below is great too.